examples/one_degree_simulation.jl

```diff
@@ -61,8 +61,8 @@ z_below_surface = r_faces[end-1]
 
 mask = LinearlyTaperedPolarMask(southern=(-80, -70), northern=(70, 90), z=(z_below_surface, 0))
 
-FT = ECCORestoring(temperature, grid; mask, rate=restoring_rate)
-FS = ECCORestoring(salinity,    grid; mask, rate=restoring_rate)
+FT = Restoring(temperature, grid; mask, rate=restoring_rate)
+FS = Restoring(salinity,    grid; mask, rate=restoring_rate)
```

Comment:
```suggestion
FS = DatasetRestoring(salinity,    grid; mask, rate=restoring_rate)
```

would this be a better name?

"dataset" is provided by context. However, "restoring" is a more general term that would also apply to an idealized restoring. As an example, we often do need to damp velocities to zero as well, but we don't need data for that.